packages/cdk/lib/stacks/nested/nextflow-engine-stack.ts

```diff
@@ -31,15 +31,15 @@ export class NextflowEngineStack extends NestedEngineStack {
     const outputBucket = Bucket.fromBucketName(this, "OutputBucket", params.outputBucketName);
     const artifactBucket = Bucket.fromBucketName(this, "ArtifactBucket", params.artifactBucketName);
 
-    const adapterRole = new Role(this, "AdapterRole", { assumedBy: new ServicePrincipal("ecs-tasks.amazonaws.com"), ...props.policyOptions });
     const engineRole = new NextflowEngineRole(this, "NextflowEngineRole", {
+      account: props.env?.account ?? "",
```

Comment:
Does this default work? If we can't figure out our region, then we won't be able to construct an ARN in the policy.

From within a stack you can use `Stack.account` and `Stack.region` to look up these values rather than passing them in via the props. https://docs.aws.amazon.com/cdk/latest/guide/stacks.html

Reply:
Thanks for pointing that out, I didn't know I can get them from within the stack, will update